### ISSUE TITLE
64 bit StandAlone

### ISSUE THREAD

**Issue by philipbaileynar** [Riverscapes/gcd#259]
Finish the work to make a 64 bit version of the GCD StandAlone. It's currently only in 32 bit.

* [x] GDAL 64 bit
* [x] Console 64 bit
* [x] Quick metrics on one large dataset

**Comment by MattReimer** [comment]
For future reference here is the secret:

```xml
  <!-- Properties group for Determining 64bit Architecture -->
  <PropertyGroup>
    <CurrentPlatform>x64</CurrentPlatform>
    <CurrentPlatform Condition="'$(Platform)'!='x64'">x86</CurrentPlatform>
  </PropertyGroup>  
  <ItemGroup>
    <Reference Include="gdalconst_csharp, Version=1.0.5387.28601, Culture=neutral, PublicKeyToken=db5a52b08dc5b321, processorArchitecture=MSIL">
      <SpecificVersion>False</SpecificVersion>
      <HintPath>..\extlib\GDAL221_$(CurrentPlatform)\csharp\gdalconst_csharp.dll</HintPath>
      <EmbedInteropTypes>False</EmbedInteropTypes>
    </Reference>
...
```

**Comment by MattReimer** [comment]
Here's another useful one (must be run in git bash cygwin terminal, not dos box)

```bash
 find -name "*.dll" | xargs file > summary.txt
```

**Comment by MattReimer** [comment]
A little preliminary but I think we're getting some small performance increase on the x64

![untitled drawing](https://user-images.githubusercontent.com/1063391/39487326-a9db71c6-4d33-11e8-953b-337aee572f4f.png)

**Comment by MattReimer** [comment]
Just fixed #262 which gives us an even greater performance boost.
